frontend/survey-project/src/app/shared/dto/responses/survey/survey-results-response.ts

```diff
@@ -0,0 +1,18 @@
+export interface SurveyResultsResponse {
+    status?: number;
+    title: string;
+    question: string;
+    ending_time: string;
+    user_ended: boolean;
+    is_anonymous: boolean;
+    responses: {
+        yes: number;
+        no: number;
+        "no response": number;
```

Comment:
Generally, Dtos should not have fields where words are divided by ' ' (space), we use camelCase or maybe dashes, but preferably camelCase.